packages/docs/content/getting-started/migrating-to-new-buttons.mdx

```diff
@@ -23,4 +23,10 @@ While we have provided **automatic migration scripts** as part of this migration
 
 ## Migration scripts
 
-Hello world
+Migration help has been provided via the `cmsds-migrate` helper script. If you include the `@cmsgov/design-system@ver` package in your project the script is usable by executing the following command from the command line:
+
+`yarn cmsds-migrate` or `npx cmsds-migrate`
+
+After executing this you will be provided with a list of migration modules to utilize which are named descriptively by function. After a configuration module is chosen, your `cwd` will be searched for files that match the pre-defined rules for migration, and a description of the changes to occur will be presented. Finally, the files to be modified will be listed, and after verification by prompt, migration will take place. **Please be sure to only run these scripts in a git repo where changes can be backed out if incorrect.**
```

Comment:
```suggestion
After executing this you will be provided with a list of migration modules to utilize which are named descriptively by function, like this:

\```
 file: 
❯ html-button-migration.mjs 
  migration-notice-cleanup.mjs 
  react-button-migration.mjs 
\```

If your project is written in React, use the `react-button-migration` option; if you use plain HTML with CSS classes, choose the `html-button-migration` option. After choosing a migration configuration, the script will search your current working directory for files that match the pre-defined rules for migration, and it will present a description of the changes to occur. Finally, it will list all the files to be modified, and after verification by prompt, it will execute the migration. **Please be sure to only run these scripts in a git repo where changes can be backed out if incorrect.** For cases that require some additional human attention, the script will place a comment near the change that it made. These comments can be cleaned up automatically using the `migration-notice-cleanup` migration configuration.
```

Note that in my suggestion, I had to escape the \``` block, which didn't really work.